packages/blade/src/components/Charts/BarChart/BarChart.web.tsx

```diff
@@ -68,19 +98,33 @@ const _ChartBar: React.FC<ChartBarProps> = ({
       animationEasing="linear"
       dataKey={dataKey}
       name={name}
-      // https://github.com/recharts/recharts/issues/2244#issuecomment-2288572842
-      // we have this shape to have distance b/w bar charts
-      // high change we might not have this in updated designs.
       shape={(props: unknown) => {
         const { fill, x, y, width, height, index: barIndex } = props as RechartsShapeProps;
         const fillOpacity = isNumber(activeIndex) ? (barIndex === activeIndex ? 1 : 0.2) : 1;
         const gap = DISTANCE_BETWEEN_STACKED_BARS;
-        const isVertical = layout === 'vertical';
+        const isVertical = orientation === 'vertical';
+
+        const getInitialOpacity = (): number => {
```

Comment:
Lets skip it for now maybe? code seems too complex for just 1 transition. We'll figure out later